README.md

```diff
@@ -238,7 +238,7 @@ jobs:
       - name: Generate Documentation
         uses: SwiftDocOrg/swift-doc@master
         with:
-          inputs: "Source"
```

Comment:
Ugh. How long has this been here? (That's purely a hypothetical question; I'm too lazy to `git blame` this right now)

I take full responsibility for this. Though if you're looking for a more impersonal agent to rage against, please direct your ire to Alamofire, which was used as my first example and remains the only library I've seen using `Source` instead of `Sources`. Actually, wait... was _I_ the one who decided that? (Again, hypothetical question)